src/docs/ganache/overview.md

```diff
@@ -12,6 +12,6 @@ layout: docs.hbs
 
 # Ganache Overview
 
-[Ganache](/ganache) is a personal blockchain for Ethereum development you can use to deploy contracts, develop your applications, and run tests.
+[Ganache](/ganache) is a personal blockchain for Ethereum and Corda development you can use to deploy contracts, develop your applications, and run tests.
```

Comment:
[Ganache](/ganache) is a personal blockchain for rapid Ethereum and Corda distributed application development. You can use Ganache across the entire development cycle; enabling you to develop, deploy and test your dApps in a safe and deterministic environment.